Source/WebKit/UIProcess/WebProcessPool.cpp

```diff
@@ -167,8 +167,6 @@ constexpr Seconds resetGPUProcessCrashCountDelay { 30_s };
 constexpr unsigned maximumGPUProcessRelaunchAttemptsBeforeKillingWebProcesses { 2 };
 #endif
 
-static constexpr Seconds audibleActivityClearDelay = 5_s;
```

Comment:
This change looks unrelated?

Reply:
There were two variables with this name in the global WebKit namespace and the unified sources shift made them conflict; so I moved this one.